src/app/components/Contact.tsx

```diff
@@ -29,11 +39,56 @@ export const Contact = () => {
         right={9}
       /> */}
       <SectionHeader>contact:</SectionHeader>
-      <ContactInfo icon="email" data="michael.gacetta@gmail.com" />
-      <ContactInfo icon="phone" data="360.791.1333" />
-      <ContactInfo icon="linkedIn" data="@gacetta" />
-      <ContactInfo icon="gitHub" data="@gacetta" />
-      <ContactInfo icon="instagram" data="@gacetta" />
+      <List rounded={4} p={4} bg={"snow"} boxShadow={"dark-lg"}>
+        <ListItem>
+          <Link
+            href="mailto:michael.gacetta@gmail.com"
+            isExternal
+            display={"flex"}
+            alignItems={"center"}
+          >
+            <EmailIcon mr={2} />
+            <Text>michael.gacetta@gmail.com</Text>
+          </Link>
+        </ListItem>
+        <ListItem display={"flex"} alignItems={"center"}>
+          <PhoneIcon mr={2} />
+          <Text>360.791.1333</Text>
+        </ListItem>
+        <ListItem>
+          <Link
+            href="https://github.com/gacetta"
```

Comment:
instead of hardcoding href here, we should put all href elsewhere and use consts instead. `href={GACETTA_GITHUB}`